### ISSUE TITLE
When today clicked, daySelect function doesn't fire.

### ISSUE THREAD

**Issue by hatimmakki** [hurkanyakay/react-weekly-day-picker#8]
Hi,

the daySelect function fires whenever the user clicks on any day in the calendar week view. but it doesn't fire when click on today.
any help with that?

this is my code
`      <ReactWeeklyDayPicker
        classNames={classNames}
        secondLineFormat={"D"}
        startDay={new Date()}
        multipleDaySelect={false}
        unselectDay={function(day) {
          console.log("unselectDay");
        }}
        onPrevClick={function(startDay, selectedDays) {
          console.log("onPrevClick");
        }}
        onNextClick={function(startDay, selectedDays) {
          console.log("onNextClick");
        }}
        selectDay={function(day) {
          console.log("selected a day", day);
        }}
        //dayCircleUnavailableText="!"
        mobilView={window.innerWidth < 1024} // enables mobil view
      />`

So, in short, how can i run a function (callback) when I select **today**. for other days, selectDay works fine, but for today it doesn't
